### PR TITLE
wrappers, tests/main/snap-service-timer: restore missing commit, add spread test for timer services

### PR DESCRIPTION
**Pull request by bboozzoo**
A spread test for timer services.

### CODE REVIEW

**Comment by bboozzoo** [comment]
Somehow this commit did not end up being merged https://github.com/bboozzoo/snapd/commit/ccb91b295d0d217ec3bc4887b8e7c349a926ec22 Will push an update shortly.

File: tests/lib/snaps/test-snapd-timer-service/meta/snap.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: test-snapd-timer-service
+version: 1.0
+apps:
+  regular-timer:
+    command: bin/loop regular-timer
+    daemon: simple
+    timer: 0:00-24:00/96
```
Comment (zyga) [inline tests/lib/snaps/test-snapd-timer-service/meta/snap.yaml]: To help casual reviewers. Is it correct to say that this means `# Run every 15 minutes.`?

File: tests/lib/snaps/test-snapd-timer-service/meta/snap.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  random-timer:
+    command: bin/loop random-timer
+    daemon: simple
+    timer: 0:00~24:00/96
```
Comment (zyga) [inline tests/lib/snaps/test-snapd-timer-service/meta/snap.yaml]: This one is `# Run roughly every 15 minutes, with (some) random distribution.`

File: wrappers/services.go [hunk 3]
```diff
@@ -560,8 +560,7 @@ X-Snappy=yes
 [Timer]
 Unit={{.ServiceFileName}}
 {{ range .Schedules }}OnCalendar={{ . }}
-{{- end }}
```
Comment (zyga) [inline wrappers/services.go]: Does this mean we used to generate a broken service file that needs to be updated?

Comment (bboozzoo) [reply wrappers/services.go]: The piece of code that did this got merged just today. On top of this, the snap won't install as systemd will reject the timer file we produced.

**Pull request merged**